### PR TITLE
Add changes for login

### PR DESCRIPTION
**Pull request by simhayoz**

### CODE REVIEW

File: app/src/main/java/ch/epfl/polychef/users/FakeConnectedActivity.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package ch.epfl.polychef.users;
+
+        import android.content.Intent;
```
Comment (Ingloryon) [inline app/src/main/java/ch/epfl/polychef/users/FakeConnectedActivity.java]: Pourquoi les imports sont indentés ?

**Review by Ingloryon — APPROVED** [review]

File: app/src/main/java/ch/epfl/polychef/HomePage.java [hunk 2]
```diff
@@ -91,26 +97,4 @@ public boolean onNavigationItemSelected(@NonNull MenuItem selectedItem) {
...
-    protected void onResume() {
-        super.onResume();
-
-        logButton.setText(LOG_OUT);
-        logButton.setOnClickListener(new View.OnClickListener() {
-            public void onClick(View view) {
-                logout(view);
-            }
-        });
-    }
```
Comment (yanickpauloamaro) [inline app/src/main/java/ch/epfl/polychef/HomePage.java:105]: What will happen when we log in again later? Will the button change to "Log Out" again?

Comment (yanickpauloamaro) [inline app/src/main/java/ch/epfl/polychef/HomePage.java:105]: If this is correct, we should do the same for the EntryPage to stay consistent.

**Review by ChapatteMateo — APPROVED** [review]

**Review by Ingloryon — APPROVED** [review]
Vraiment ouf comment ca peut etre utile mockito :o

**Pull request merged**